openapi.yaml

```diff
@@ -2851,6 +2851,20 @@ components:
           $ref: '#/components/schemas/period-start'
         values:
           $ref: '#/components/schemas/limit-value-set'
+        conditional-values:
+          type: array
+          description: |
+            List of alternative ratings for rating altering conditions on the segment.  References condition identifiers described in
+            `getNetworkSegment`
```

Comment:
Based on this comment, I believe the intent for `condition` is to be a `#/components/schemas/network-component-name` not a `#/components/schemas/generic-identifier` as below.

Reply:
Right you are, fixed.